pkg/component/observability/monitoring/prometheus/garden/scrapeconfigs.go

```diff
@@ -25,3 +35,69 @@ var cAdvisor string
 func AdditionalScrapeConfigs() []string {
 	return []string{cAdvisor}
 }
+
+// CentralScrapeConfigs returns the central ScrapeConfig resources for the garden prometheus.
+func CentralScrapeConfigs(prometheusAggregateTargets []monitoringv1alpha1.Target, globalMonitoringSecret *corev1.Secret) []*monitoringv1alpha1.ScrapeConfig {
+	out := []*monitoringv1alpha1.ScrapeConfig{{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "prometheus",
+		},
+		Spec: monitoringv1alpha1.ScrapeConfigSpec{
+			StaticConfigs: []monitoringv1alpha1.StaticConfig{{
+				Targets: []monitoringv1alpha1.Target{"localhost:9090"},
+			}},
+			RelabelConfigs: []*monitoringv1.RelabelConfig{{
+				Action:      "replace",
+				Replacement: "prometheus-garden",
+				TargetLabel: "job",
+			}},
+			MetricRelabelConfigs: monitoringutils.StandardMetricRelabelConfig("prometheus_(.+)"),
+		},
+	}}
+
+	if globalMonitoringSecret != nil {
+		// if len(prometheusAggregateTargets) > 0 && globalMonitoringSecret != nil {
+		out = append(out, &monitoringv1alpha1.ScrapeConfig{
+			ObjectMeta: metav1.ObjectMeta{Name: "prometheus-" + aggregate.Label},
+			Spec: monitoringv1alpha1.ScrapeConfigSpec{
+				HonorLabels:     ptr.To(true),
+				HonorTimestamps: ptr.To(false),
+				MetricsPath:     ptr.To("/federate"),
+				Scheme:          ptr.To("HTTPS"),
+				Params: map[string][]string{
+					"match[]": {
+						`{__name__=~"seed:(.+):count"}`,
+						`{__name__=~"seed:(.+):sum"}`,
+						`{__name__=~"seed:(.+):sum_cp"}`,
+						`{__name__=~"seed:(.+):sum_by_pod",namespace=~"extension-(.+)"}`,
+						`{__name__=~"shoot:(.+):(.+)",__name__!="shoot:apiserver_storage_objects:sum_by_resource",__name__!="shoot:apiserver_watch_duration:quantile"}`,
+						`{__name__="ALERTS"}`,
+						`{__name__="shoot:availability"}`,
+						`{__name__="prometheus_tsdb_lowest_timestamp"}`,
+						`{__name__="prometheus_tsdb_storage_blocks_bytes"}`,
+						`{__name__="seed:persistentvolume:inconsistent_size"}`,
+						`{__name__="seed:kube_pod_container_status_restarts_total:max_by_namespace"}`,
+						`{__name__=~"metering:.+:(sum_by_namespace|sum_by_instance_type)"}`,
+					},
+				},
+				TLSConfig: &monitoringv1.SafeTLSConfig{InsecureSkipVerify: true},
```

Comment:
Because the scrape config targets all aggregate Prometheis running on all seeds, and their server certs are not signed by the same CA/we don't even know the CA.

Reply:
It is unfortunate security-wise, but I understand that this will be difficult to change and would introduce new dependencies, which we might not want.